lib/src/configcat_client.dart

```diff
@@ -82,14 +94,20 @@ class ConfigCatClient {
 
   /// Gets the value of a feature flag or setting as [T] identified by the given [key].
   ///
-  /// [key] is the identifier of the feature flag or setting.
-  /// In case of any failure, [defaultValue] will be returned.
-  /// [user] is the user object to identify the caller.
+  /// [key] the identifier of the feature flag or setting.
+  /// [defaultValue] in case of any failure, this value will be returned.
+  /// [user] the user object.
+  /// [T] the type of the desired feature flag or setting. Only [String], [int], [double], [bool] or [dynamic] types are supported.
```

Comment:
Let's match the .NET SDK's intellisense doc which has already passed the UX review.

```suggestion
  /// [T] the type of the desired feature flag or setting. Only the following types are allowed: [String], [bool], [int], [double], [Object] (both nullable and non-nullable) and [dynamic].
```